{{ cookiecutter.repo_name }}/Makefile

```diff
@@ -51,3 +52,13 @@ docs: ## Build docs using Sphinx and copy to docs folder (this makes it easy to
 
 ipython: ## Provides an interactive ipython prompt
 	docker exec -it $(CONTAINER_NAME) ipython
+
+setup-dvc: ## Initializes DVC in project repo
+	docker exec -it $(CONTAINER_NAME) dvc init
+	git add .dvc .dvcignore
+	git commit -m "Initialize DVC in repo"
+
+dvc-remote: ## Adds DVC remote to repo.
```

Comment:
Does this depend on`dev-setup`?

Reply:
Do you mean `setup-dvc`? If so then yes. I like you're suggestion of moving the commands under `setup-dvc` to `hooks/post_gen_project.sh`.

Reply:
Cool!